client-v2/src/components/FrontsEdit/CollectionComponents/CollectionItem.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import { connect } from 'react-redux';
```

Comment:
Formerly ArticleFragment, this now represents anything we might want to render in a collection.